table.scss

```diff
@@ -0,0 +1,126 @@
+@import 'bower_components/vui-colors/colors.scss';
+@import 'bower_components/vui-typography/typography.scss';
+@import 'bower_components/vui-typography/px-to-base-rem.scss';
+
+@mixin vui-table() {
+	background-color: #ffffff;
+	border: 1px solid $vui-color-titanius;
+	border-collapse: separate;
+	border-radius: 6px;
+	border-spacing: 0;
+	display: table;
+	width: 100%;
+
+	& thead {
+		display: table-header-group;
+
+		& th {
+			@include vui-table-header-cell();
+		}
+	}
+
+	& tfoot {
+		& tr:first-child td {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		& tr:first-child th {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		display: table-footer-group;
+	}
+
+	& tr {
+		display: table-row;
+	}
+
+	& td {
+		@include vui-table-cell();
+	}
+
+	& tr td:last-child {
+		border-right: none;
+	}
+
+	& tr:last-child td {
+		border-bottom: none;
+	}
+
+	& tbody th,
+	  tfoot th {
```

Comment:
Could this rule be combined with the one above for `& td`? They both just call the `vui-table-cell()` mixin.